feature-configs/5g-ref-nw/clusters/otwaon34567sno/nw-config/sriov/sriov-networks.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriov-nw-du-fec-fh
+  namespace: openshift-sriov-network-operator
+spec:
+  vlan: 142
+  resourceName: du_fec_fh_otwaon34567sno
+---
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriov-nw-du-fec-mh-c
+  namespace: openshift-sriov-network-operator
+spec:
+  vlan: 100
+  resourceName: du_fec_mh_otwaon34567sno
+---
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
```

Comment:
Need to change the API.  Note that we also need to deal with different operators for Mt Bryce vs N3000.